angular-poller.js

```diff
@@ -308,6 +308,19 @@
                 reset: function () {
                     this.stopAll();
                     pollers = [];
+                },
+
+                /**
+                * Stop and removes a specific poller service
+                */
+                remove: function (poller) {
```

Comment:
couldn't see a way to remove it from the pollers array then, open to suggestions :)